### DISCUSSION TITLE
Looking for the perfect template

### DISCUSSION THREAD

**Discussion by DavideC03** [github/india#69] [GitHub Platform]

**Comment by rohitnb** [comment]
Hi @DavideC03, I'm assuming that you're talking about repository structures for a project. If not, please add more details on what kind of template you're thinking of. 

Typically though, depending on the project type (Node, .NET, etc.), there maybe multiple templates you could use to achieve what you need to do. In my honest opinion, there isn't anything like a perfect one-shoe-fits-all template out there and it totally depends on the requirements of your project. There are standard templates that you can explore GitHub to find, or use scaffolding techniques offered by multiple project ecosystems. 

After your research, if you find a really cool project structure that works for you, you could consider turning that into a template for your future projects quite easily using GitHub. Once a template repo is created, you can create copies of it in GitHub super easily. For more info, take a look at this [doc page](https://www.geeksforgeeks.org/scaffolding-expressjs-app-scratch/).

**Reply by DavideC03** [reply]
I'm searching a sort of a default template, that is not optimized for any work, but can work quiet well for every work.
From this template I want to create all others (specific) templates.

**Comment by FRTMentor** [comment]
Please specify the proper requirement to get the related templets.
